### DISCUSSION TITLE
Content of geojson

### DISCUSSION THREAD

**Discussion by hski-github** [neocarto/bertin#72] [Q&A] [answered]
I understand, bertin needs a map as a source as parameter geojson as a kind of base map. Then I see https://raw.githubusercontent.com/neocarto/bertin/main/data/world.geojson being used in the examples. This contains countries with name and ISO3 code and their shape, right? Do you have an easy way to list, which countries are included?

Guess there are other geojson with more detailed information like states in the US or postal code regions worldwide? Are there geojson sources you can recommend or worth to review? You know the community I guess. :-)

Could be helpful for the README.md to have some links for those how would need other maps.

**Comment by neocarto** [comment, marked as answer]
To have the liste of the countries you can write this line. It's in js: 
world.features.map((d) => d.properties.NAMEen)
There are many other resources to retrieve geometries.
https://www.naturalearthdata.com/ or https://ec.europa.eu/eurostat/fr/web/gisco/geodata/reference-data
